src/platforms/macos/daemon/wireguardutilsmacos.cpp

```diff
@@ -265,10 +266,10 @@ bool WireguardUtilsMacos::deleteRoutePrefix(const IPAddress& prefix) {
   if (!m_rtmonitor) {
     return false;
   }
+
   if (prefix.prefixLength() > 0) {
-    return m_rtmonitor->insertRoute(prefix);
+    return m_rtmonitor->deleteRoute(prefix);
```

Comment:
Was this a separate bug?

Reply:
Yes, I think this was a bug. The impact of the bug was small because the routes are mostly automatically deleted upon interface destruction anyways - but it could result in zombie routes in the table if the interface doesn't go down (eg: during a silent server switch).